sdk/eventhub/azure-eventhub/azure/eventhub/aio/_consumer_async.py

```diff
@@ -182,7 +183,8 @@ def _next_message_in_buffer(self):
         # pylint:disable=protected-access
         message = self._message_buffer.popleft()
         event_data = EventData._from_message(message)
-        event_data._uamqp_message = message
+        if self._amqp_transport != PyamqpTransportAsync:
```

Comment:
once Swathi's PR goes through, you'll have to update this to use KIND

Reply:
Shouldn't this be `isinstance()`?
(And when we do update to `KIND` we can remove the import statement)